### PR TITLE
chore(rrweb): bump rrweb version to 2.22.0

### PR DESCRIPTION
**Pull request by michellewzhang**
another version to release bug fixes introduced in https://github.com/getsentry/rrweb/pull/202

https://github.com/getsentry/publish/issues/3968

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -11709,6 +11731,11 @@ typescript@^5.3.3, typescript@^5.4.5:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-5.4.5.tgz#42ccef2c571fdbd0f6718b1d1f5e6e5ef006f611"
   integrity sha512-vcI4UpRgg81oIRUFwR0WSIHKt11nJ7SAVlYNIu+QpqeyXP+gpQJy/Z4+F0aGxSE4MqwjyXvW/TzgkLAx2AGHwQ==
 
+typescript@next:
+  version "5.6.0-dev.20240613"
```
Comment (scttcper) [inline yarn.lock:11735]: this is odd

Comment (michellewzhang) [reply yarn.lock:11735]: i just ran `npx yarn-deduplicate && yarn install` and it happened

File: yarn.lock [hunk 2]
```diff
@@ -3095,44 +3078,49 @@
...
-    "@sentry-internal/rrdom" "2.18.0"
-    "@sentry-internal/rrweb-snapshot" "2.18.0"
-    "@sentry-internal/rrweb-types" "2.18.0"
+    "@sentry-internal/rrdom" "2.20.0"
+    "@sentry-internal/rrweb-snapshot" "2.20.0"
+    "@sentry-internal/rrweb-types" "2.20.0"
     "@types/css-font-loading-module" "0.0.7"
     "@xstate/fsm" "^1.4.0"
     base64-arraybuffer "^1.0.1"
+    downlevel-dts "https://github.com/getsentry/downlevel-dts"
```
Comment (scttcper) [inline yarn.lock:3123]: sentry-internal/rrweb should probably move downlevel-dts to dev dependencies, doesn't seem like something everyone needs to install

Comment (scttcper) [reply yarn.lock:3123]: cc @billyvg

Comment (michellewzhang) [reply yarn.lock:3123]: https://github.com/getsentry/rrweb/pull/204

**Review by scttcper — APPROVED** [review]

**Pull request merged**